DemoApp/StreamChat/StreamChatWrapper+DemoApp.swift

```diff
@@ -23,7 +19,18 @@ extension StreamChatWrapper {
         if client == nil {
             client = ChatClient(config: config)
         }
-        client?.registerAttachment(LocationAttachmentPayload.self)
+
+        // Custom Attachments
+        if AppConfig.shared.demoAppConfig.isLocationAttachmentsEnabled {
```

Comment:
I think it should be always on, since it's the demo app. Our config screen becomes very complex at this point.